docs/docs/features/dongle.mdx

```diff
@@ -0,0 +1,349 @@
+You will get an error message similar to `zmk: No behavior assigned to 7 on layer 0`.
+:::
+
+### Setting user configuration
+
+Now that we have defined the default shield configuration, we can easily enable the dongle in the user configuration.
+If we do not want to use the dongle, this step can be skipped.
+
+
+<Tabs
+defaultValue="split"
+values={[
+{label: 'Split Keyboard', value: 'split'},
+{label: 'Unibody Keyboard', value: 'unibody'},
+]}>
+
+<TabItem value="split">
+
+```kconfig title="config/my_board_left.conf"
+# Disable central role to use dongle
+CONFIG_ZMK_SPLIT_ROLE_CENTRAL=n
+```
+
+When using a split keyboard, you can use a single file without the `_left` or `_right` suffix to configure both sides. 
+For example, `corne.conf` and `corne.keymap` will apply to both `corne_left` and `corne_right`. If a shared config file exists, any left or right files will be ignored.
+
+</TabItem>
+<TabItem value="unibody">
+
+```kconfig title="config/my_board.conf"
+# Enable split functionality, this will put the unibody keyboard in peripheral mode
+CONFIG_ZMK_SPLIT=y
+```
+</TabItem>
+</Tabs>
+
+:::note
+More information about [User Config Folder](../config/index.md#user-config-folder)
+:::
+
+### Building the firmware
+
+<Tabs
+defaultValue="split"
+values={[
+{label: 'Split Keyboard', value: 'split'},
+{label: 'Unibody Keyboard', value: 'unibody'},
+]}>
+
+<TabItem value="split">
+After writing the configuration files, you can modify the `build.yml` file to include the dongle configuration.
+
+Please keep in mind that the dongle does not have to be the same microcontroller as the keyboard halves.
+Any microcontroller can be used as the dongle that supports BLE.
+
+```yaml
+include:
+  - board: nice_nano_v2
+    shield: my_board_left
+  - board: nice_nano_v2
+    shield: my_board_right
+  - board: nice_nano_v2
+    shield: my_board_dongle
+  - board: nice_nano_v2
+    shield: settings_reset
+```
+
+Start by flashing the `settings_reset` firmware on all the devices.
+
+:::note
+More information about reseting split keyboards can be found under [troubleshooting Connection Issues](../troubleshooting/connection-issues.mdx#reset-split-keyboard-procedure)
+:::
+</TabItem>
+<TabItem value="unibody">
+After writing the configuration files, you can modify the `build.yml` file to include the dongle configuration.
+
+Please keep in mind that the dongle does not have to be the same microcontroller as the unibody keyboard microcontroller.
+Any microcontroller can be used as the dongle that supports BLE.
+
+```yaml
+include:
+  - board: nice_nano_v2
+    shield: my_board
+  - board: nice_nano_v2
+    shield: my_board_dongle
+  - board: nice_nano_v2
+    shield: settings_reset
+```
+
+:::note
+More information about troubleshooting keyboards can be found under [troubleshooting Connection Issues](../troubleshooting/connection-issues.mdx#reset-split-keyboard-procedure)
+:::
+
+</TabItem>
+</Tabs>
```

Comment:
Again, I think we can cut down on text and tabs by simply saying to *add* the lines to your build.yaml.